test/unit/configuration_test.rb

```diff
@@ -34,4 +39,21 @@ def test_warn_when_temporarily_changes_config_when_given_block
     assert yielded
     assert_equal :allow, Mocha.configuration.stubbing_method_unnecessarily
   end
+
+  def test_strict_keyword_argument_matching_works_is_false_by_default
+    assert_equal false, Mocha.configuration.strict_keyword_argument_matching?
+  end
+
+  if Mocha::RUBY_V27_PLUS
+    def test_enabling_strict_keyword_argument_matching_works_in_ruby_2_7_and_above
+      Mocha.configure { |c| c.strict_keyword_argument_matching = true }
+      assert_equal true, Mocha.configuration.strict_keyword_argument_matching?
```

Comment:
Minor: I'd prefer to do `assert Mocha.configuration.strict_keyword_argument_matching?` here.